pages/admin/site/moderation.vue

```diff
@@ -0,0 +1,377 @@
+                  <DataserviceBadge
+                    v-else-if="subjects[report.id].type === 'Dataservice'"
+                    :dataservice="subjects[report.id].value"
+                  />
+                  <ReuseBadge
+                    v-else-if="subjects[report.id].type === 'Reuse'"
+                    :reuse="subjects[report.id].value"
+                  />
+                  <div v-else>
+                    —
+                  </div>
+                </div>
+                <AdminBadge
+                  v-else
+                  type="danger"
+                  size="xs"
+                >
+                  {{ $t('Supprimé') }}
+                </AdminBadge>
+              </td>
+              <td>
+                <div v-if="subjects[report.id] && subjects[report.id].value">
+                  <div v-if="'created_at' in subjects[report.id].value">
+                    {{ formatDate(subjects[report.id].value.created_at) }}
+                  </div>
+                  <div v-else-if="'created' in subjects[report.id].value">
+                    {{ formatDate(subjects[report.id].value.created) }}
+                  </div>
+                  <div v-else>
+                    —
+                  </div>
+                  <p
+                    v-if="report.subject && report.subject.id in activities"
+                    class="inline-flex items-center"
+                  >
+                    {{ t('par ') }}
+                    <AvatarWithName
+                      class="fr-ml-1v"
+                      :user="activities[report.subject.id].actor"
+                    />
+                  </p>
+                </div>
+              </td>
+              <td>
+                <div>{{ formatDate(report.reported_at) }}</div>
+                <p
+                  v-if="report.by"
+                  class="inline-flex items-center"
+                >
+                  {{ t('par ') }}
+                  <AvatarWithName
+                    class="fr-ml-1v"
+                    :user="report.by"
+                  />
+                </p>
+              </td>
+              <td>
+                <div class="font-bold">
+                  {{ reasons.find((reason) => reason.value === report.reason)?.label || report.reason }}
+                </div>
+                {{ report.message }}
+              </td>
+              <td>
+                <div
+                  v-if="subjects[report.id] && subjects[report.id].value"
+                  class="flex items-center"
+                >
+                  <BrandedButton
+                    size="xs"
+                    color="tertiary"
+                    type="button"
+                    :icon="RiCheckLine"
+                    icon-only
+                    keep-margins-even-without-borders
+                    :disabled="!!report.dismissed_at"
+                    @click="dismiss(report)"
+                  >
+                    {{ $t('Rejeter le signalement') }}
```

Comment:
The labels don't show at the moment

Reply:
It's an `icon-only` button, the label is only here for accessibility reasons

Reply:
Could we have it on hover as well?